packages/client/src/runtime/core/model/applyModel.ts

```diff
@@ -34,14 +35,20 @@ const aggregateProps = ['aggregate', 'count', 'groupBy'] as const
 export function applyModel(client: Client, dmmfModelName: string) {
   // we use the javascript model name for display purposes
   const jsModelName = dmmfToJSModelName(dmmfModelName)
+  const model = client._baseDmmf.modelMap[dmmfModelName]
+  const fieldsProxyEnabled = client._engineConfig.previewFeatures?.includes('fieldReference')
   const ownKeys = getOwnKeys(client, dmmfModelName)
   const baseObject = {} // <-- user mutations go in there
+  let fieldsProxy: FieldProxy | null = null
```

Comment:
```suggestion
  let fieldsProxy: FieldProxy | undefined
```
I try to always avoid null, is there a specific reason in this case?